packages/docs/pages/users/governance/governance-actions.mdx

```diff
@@ -0,0 +1,108 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Governance actions
+
+## Submitting the proposal
+While in the same directory as your final `proposal.json` file, you can submit the proposal with:
+
+**For a default proposal:**
+
+```shell copy
+namada client init-proposal --data-path proposal.json 
+```
+
+**For non-default proposals:**
+
+One of the flags `--pgf-stewards`, `--pgf-funding`, `--eth` must be specified. For example, for a PGF steward proposal:
+
+```shell copy
+namada client init-proposal --pgf-stewards --data-path proposal.json
+```
+
+## Query the proposal
+If the submitted transaction was accepted, the user can query all the proposals with:
```

Comment:
Instead of all, I think the client only displays the 10 most recent proposals

Reply:
thx fixed in 251a337a641d60c0adfa3cc96969fc044f8d7ef8